services/Cargo.toml

```diff
@@ -107,17 +107,18 @@ tracing-subscriber = { version = "0.3", features = [
 ] }
 typetag = "0.2"
 url = { version = "2.2", features = ["serde"] }
-utoipa = { version = "3.0", features = ["actix_extras", "chrono", "uuid"] }
-utoipa-swagger-ui = { version = "3.1", features = ["actix-web"] }
+utoipa = { git = "https://github.com/1lutz/utoipa.git", features = ["actix_extras", "chrono", "uuid", "debug"] }
+utoipa-swagger-ui = { git = "https://github.com/1lutz/utoipa.git", features = ["actix-web"] }
```

Comment:
Since the PR is merged, we should at least point to the utoipa repo and add a TODO here:
https://github.com/juhaku/utoipa/pull/752

Reply:
Das debug Feature kann man wieder entfernen. Das wird nicht mehr benutzt.